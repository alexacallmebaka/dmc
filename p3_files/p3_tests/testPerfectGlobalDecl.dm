a : perfect int;b: perfect bool;
