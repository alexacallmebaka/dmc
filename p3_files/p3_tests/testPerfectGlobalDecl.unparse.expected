a : perfect int;
b : perfect bool;
c : perfect void;
